include/rocksdb/options.h

```diff
@@ -1864,6 +1864,10 @@ struct TraceOptions {
   uint64_t sampling_frequency = 1;
   // Note: The filtering happens before sampling.
   uint64_t filter = kTraceFilterNone;
+  // When true, the order of write records in the trace will match the order of
+  // the corresponding write records in the WAL and applied to the DB. There may
+  // be a performance penalty associated with preserving this ordering.
+  bool preserve_write_order = false;
```

Comment:
Maybe add the comments to explain the default behavior: the trace record will not match the order to write records in the WAL.

Reply:
Done.